cmd/goat/account_migrate.go

```diff
@@ -235,7 +234,37 @@ func runAccountMigrate(cctx *cli.Context) error {
 
 	signedPlcOpResp, err := agnostic.IdentitySignPlcOperation(ctx, oldClient, &unsignedOp)
 	if err != nil {
-		return fmt.Errorf("failed requesting PLC operation signature: %w", err)
+		slog.Warn("failed signing PLC operation", "err", err)
+		confirmPrompt := promptui.Prompt{
```

Comment:
this should be conditional on a check of whether the process is connected to a live terminal or not